src/iop/colorbalance.c

```diff
@@ -187,6 +161,62 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
   }
 }
 
+#if defined(__SSE__)
+void process_sse2(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const void *const ivoid,
+             void *const ovoid, const dt_iop_roi_t *const roi_in, const dt_iop_roi_t *const roi_out)
+{
+  dt_iop_colorbalance_data_t *d = (dt_iop_colorbalance_data_t *)piece->data;
+  const int ch = piece->colors;
+
+  // these are RGB values!
+  const __m128 lift = { 2.0 - (d->lift[CHANNEL_RED] * d->lift[CHANNEL_FACTOR]),
```

Comment:
I'm not 100% sure whether such initialization of `__m128` not using `_mm_set_ps()` is standard

Reply:
And I missed that one, too. I fixed it in other places, but here I left it in. Thanks for noticing.